### ISSUE TITLE
Изменение общей концепции работы бота

### ISSUE THREAD

**Issue by Chiliec** [GolosTools/golos-vote-bot#55]
В связи с увеличением количества желающих поддержки и неработающей системой отсева "плохого" контента, предлагаю:
- [ ] Отделить очередь голосования от голосов кураторов
- [ ] Добавить запрет на предложение постов для тех, чьи посты кураторы часто отклоняют (для снижения нагрузки на кураторов)
- [ ] В получившейся очереди голосовать за собравший наибольшее количество лайков раз в **36 минут**. *Если выплата к моменту голосования уже произведена, сразу же выбирать следующий пост.*
- [ ] Перенести голосование из группы в самого бота (для исключения внешнего давления на кураторов)
- [ ] Для того чтобы стать куратором, сделать обязательным прочтение текста с объяснением принципов курирования
- [ ] Вознаграждение самых активных кураторов (#13) из GBG, полученных с отчетов (#24)

**Comment by esper0s** [comment]
"Отделить очередь голосования от голосов кураторов"
Поясни плиз, что ты имеешь в виду
"Добавить запрет на предложение постов для тех, чьи посты кураторы часто отклоняют (для снижения нагрузки на кураторов)"
Поддерживаю, только реализовать это в виде понижения репы до определенного значения, при котором невозможно будет поставить пост на голосование, ну и реализовать возможность реабилитации
"В получившейся очереди голосовать за собравший наибольшее количество лайков раз в 36 минут. Если выплата к моменту голосования уже произведена, сразу же выбирать следующий пост."
А остальные посты мимо кассы? Как-то не комильфо
"Перенести голосование из группы в самого бота (для исключения внешнего давления на кураторов)"
Тут тоже не понял - как это будет выглядеть? Приведи пример плиз
"Для того чтобы стать куратором, сделать обязательным прочтение текста с объяснением принципов курирования"
Здравая мысль
Вознаграждение самых активных кураторов (#13) из GBG, полученных с отчетов (#24)
мысль интересная, но за кол-во дизлайков давать плюшки ИМХО перебор, начнут тупо всем вваливать дизлайки.

**Comment by Chiliec** [comment]
> Поясни плиз, что ты имеешь в виду

сейчас голосование происходит после набора определенного количества голосов, а будет  очередь и время, через которое будет извлекаться топовый пост из неё

> Поддерживаю, только реализовать это в виде понижения репы до определенного значения, при котором невозможно будет поставить пост на голосование, ну и реализовать возможность реабилитации

От репы думаю совсем отказаться, смысла в ней больше не вижу. Реабилитация может быть просто временем, проверка за последние n дней.

> А остальные посты мимо кассы? Как-то не комильфо

Что предлагаешь? При текущем подходе просто сольём батарейки в ноль поддерживая всё и не поддержим в итоге никого. Ну и всякий шлак, получающий сейчас голоса, перестанет их получать.

> Тут тоже не понял - как это будет выглядеть? Приведи пример плиз

Бот будет отправлять такое сообщение, которое отправляет в группу, в личку каждому куратору.

> мысль интересная, но за кол-во дизлайков давать плюшки ИМХО перебор, начнут тупо всем вваливать дизлайки.

Там следует читать второе сообщение, это моя текущая позиция.

**Comment by esper0s** [comment]
> А остальные посты мимо кассы? Как-то не комильфо
> 
> Что предлагаешь? При текущем подходе просто сольём батарейки в ноль поддерживая всё и не поддержим в итоге никого. Ну и всякий шлак, получающий сейчас голоса, перестанет их получать.
Ну с расчетом на поддержку годного контента да, правильно, только расчитать кол-во апов в день.

> мысль интересная, но за кол-во дизлайков давать плюшки ИМХО перебор, начнут тупо всем вваливать дизлайки.
> 
> Там следует читать второе сообщение, это моя текущая позиция.
Тогда стоит добавить возможность комментировать дизлайк, чтобы было видно, что человек не просто так его поставил

> Поддерживаю, только реализовать это в виде понижения репы до определенного значения, при котором невозможно будет поставить пост на голосование, ну и реализовать возможность реабилитации
> 
> От репы думаю совсем отказаться, смысла в ней больше не вижу. Реабилитация может быть просто временем, проверка за последние n дней.
В чем будет выражаться проверка? и сколько дней?
Ну и в итоге - что получит пост, признанный годным? Все тот же паровозик из апов членов сообщества? Но при том, что с изменениями многие будут несогласны, он заметно уменьшится, будут ли еще какие-то доп. вливания в СГ паровозика, либо это будет что-то типа делегированной СГ, для более весомого апа?

**Comment by Chiliec** [comment]
> Тогда стоит добавить возможность комментировать дизлайк, чтобы было видно, что человек не просто так его поставил

Ну поставил и поставил, найдутся другие кто лайки поставит :) Хотя, посмотрим...

> В чем будет выражаться проверка? и сколько дней?

Процент отклонённых постов за n дней, где n можно будет поменять в настройках)

> Все тот же паровозик из апов членов сообщества? 

Да

> Но при том, что с изменениями многие будут несогласны, он заметно уменьшится?

Возможно уменьшится количественно, но я почти не сомневаюсь что пополнится качественно 🐳

> будут ли еще какие-то доп. вливания в СГ паровозика, либо это будет что-то типа делегированной СГ, для более весомого апа?

Вряд ли, такого пока просто нет в Голосе.

**Comment by esper0s** [comment]
> будут ли еще какие-то доп. вливания в СГ паровозика, либо это будет что-то типа делегированной СГ, для более весомого апа?
> 
> Вряд ли, такого пока просто нет в Голосе.
может не так выразился - я имел в виду что-то вроде доброго кита

**Comment by Chiliec** [comment]
Нет, мне эти заморочки не сильно интересны и чужие деньги не хочу хранить у себя.

**Comment by esper0s** [comment]
я тут еще что подумал - для кураторов надо будет сделать кнопку типа показать позже. Например пост вроде бы и неплохой, но на однозначный лайк не тянет, но и дизлайк не заслужил. а зная, что  нужно будет выбрать еще Х штук постов, он его как-то отметит и перейдет к следующему. Ну а потом уже примет решение

**Comment by worthless-man** [comment]
Я думаю, стоит голосовать 35 раз в день, а 5 голосов оставить пользователям для личных нужд

**Comment by esper0s** [comment]
> Я думаю, стоит голосовать 35 раз в день, а 5 голосов оставить пользователям для личных нужд
Да я думаю еще меньше - раз 20. Но 100% и сделать это обязательным условием. либо больше, но с меньшим процентом.

**Comment by Chiliec** [comment]
> но с меньшим процентом

Тогда новички будут за бортом, не хотелось бы. Они не могут голосовать меньше 100%

**Comment by esper0s** [comment]
тогда надо брать качеством - 20 постов и все голосуют со 100%. Мне например 35 уже много - по 20апов я выставляю на бирже

**Comment by esper0s** [comment]
А вообще. мне кажется, что нужно было немного не так менять концепцию. Не в техническом смысле, а в организационном. Если ты планируешь привлечь  дельфинов/китов/касаток, то было бы неплохо этим сначала озаботиться, а потом уже вместе с ними менять концепцию. Я уверен, у них тоже найдутся свои пожелания и предпочтения, для многих из них даже 20 апов будет много

**Comment by Chiliec** [comment]
> тогда надо брать качеством - 20 постов 

да, раз в час => 24 постов в сутки кажется самым нормальным решением :)
> Если ты планируешь привлечь дельфинов/китов/касаток

планирую, поэтому и не будет
> и все голосуют со 100%

**Comment by esper0s** [comment]
ну тогда представляй нам скорее голосовалочку 2.0 или какая она там будет?))

**Comment by Chiliec** [comment]
Не знаю, 1.0 ещё даже не было. Сырущая альфа :)

**Comment by esper0s** [comment]
главное. что люди оценили, ну и ты осознал куда и как двигаться)
